src/client/AddMealComponents/AddMeal.js

```diff
@@ -0,0 +1,121 @@
+import React from "react";
+import "./form.css";
+import { useState } from "react";
```

Comment:
```suggestion
import { useState } from "react";
import "./form.css";
```

Try to sort your imports in order of how "far away" they come from. 
Third party imports, like from node_modules go at the top.
Then anything from other folders and finally things in the same folder.